test/fixtures/cookbooks/kubernetes-stack-test/recipes/kubectl_install_default.rb

```diff
@@ -1 +1,3 @@
-kubectl 'install default kubectl'
+kubectl 'install default kubectl' do
```

Comment:
should have 2 cases:
- install default kubectl
- install specific kubectl version